src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        Ball ball = new Ball();
+        ball.setNumber(random.nextInt(100));
```

Comment:
magic number

Reply:
not fixed

Reply:
![image](https://user-images.githubusercontent.com/12864128/220138889-2329666e-3843-41f4-ae2b-5f25baf0c75a.png)